cloudferrylib/os/actions/check_sql.py

```diff
@@ -13,10 +13,23 @@
 # limitations under the License.
 
 from cloudferrylib.base.action import action
+from cloudferrylib.os.compute import nova_compute
+from cloudferrylib.os.image import glance_image
+from cloudferrylib.os.storage import cinder_database
+from cloudferrylib.utils import utils as utl
```

Comment:
I guess we can use full name here (`utils`) - there is no need to save space or memory here.